doc-site/docs/faqs.md

```diff
@@ -1,22 +1,96 @@
+
+* **Cost:** Running a private transaction involves off-chain computation and settles as a smaller, simpler transaction on the base ledger. This typically leads to significantly lower and more predictable gas fees compared to executing complex logic entirely on-chain.
+* **Performance:** Paladin's architecture is a generational leap forward. Instead of running a full, persistent private blockchain, its `pente` domain spins up an ephemeral **Besu EVM** in memory just to execute a single transaction. This is a far more scalable and efficient model.
+
+### What programming languages do I use to build on Paladin?
+
+**Developers use Solidity for on-chain smart contracts and any standard language for off-chain client applications.**
+
+* For the **smart contracts** that define your assets and logic within a Domain, you use languages that compile to the **EVM**, primarily **Solidity** or Vyper.
+* For **client-side applications** that interact with the Paladin runtime, you can use any language that can communicate via a standard JSON-RPC API to send transactions and query state.
+
+### How do I get started with a local setup?
+
+**You can get a local Paladin network running quickly using the official quick start guide.** This process uses a Kubernetes operator to automatically deploy all the necessary components, including a private Besu blockchain and a three-node Paladin network, right on your laptop.
+
+You can find the complete, step-by-step instructions in the **[Installation Guide](https://lf-decentralized-trust-labs.github.io/paladin/head/getting-started/installation/)**.
+
+### Where is the best place to ask questions or get help?
+
+**The best place to connect with the Paladin community and development team is on the Hyperledger Discord server.** This is the primary forum for real-time discussions, questions, and sharing ideas.
+
+* Join the conversation here: **[Hyperledger Discord](https://discord.gg/hyperledger)** (look for the Paladin channel).
+
+### How should I report a bug?
+
+**Bugs should be reported by creating a new issue in the official GitHub repository.** When reporting a bug, please provide as much detail as possible, including your operating system, project version, and clear steps to reproduce the issue.
+
+* Report a bug here: **[Paladin GitHub Issues](https://github.com/LF-Decentralized-Trust-labs/paladin/issues)**
+
+### What are the guidelines for contributing to Paladin?
+
+**Contributions from the community are highly encouraged and welcome.** The general process involves forking the repository, creating a feature branch for your changes, and submitting a pull request for review. All code contributions must include tests and a "Signed-off-by" line in the commit message to comply with the Developer Certificate of Origin (DCO).
+
+* Explore the code and contribute here: **[Paladin GitHub Repository](https://github.com/LF-Decentralized-Trust-labs/paladin)**
```

Comment:
It might be good to add a contributing file such as https://github.com/hyperledger/firefly/blob/main/CONTRIBUTING.md and then reference it here.

In fact more specifically we should probably review everything in https://github.com/LF-Decentralized-Trust/governance/tree/main/tac/guidelines and start aligning the project to those guidelines. Happy for this PR to merge without that but wanted to mention.